internal/filesystem/linux_filesystem.go

```diff
@@ -0,0 +1,46 @@
+package filesystem
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+type LinuxFilesystem struct{}
```

Comment:
There are codes that specific to linux and give name with OS would be helpful in case other contributor (hopefully) notice that our app doesnt support windows.